agent/tool-scripts/tests/lockstat/gold/stderr

```diff
@@ -2,11 +2,18 @@ lockstat: Error, one of the following options is required, --install|--start|--s
 
 The following options are available:
 
+Operational options - do *not* specify any of these when you register a tool:
+
 	--install                 verify this perf tool will work
 
 	--start|stop|postprocess  start/stop/postprocess the data collection
 	-d str|--dir=str          directory to store data collection (required)
 
+Tool-specific options - these should be specified when you register a tool:
+
+
+Help option - do *not* specify this when you register a tool:
```

Comment:
This looks funny...should we tweak the `lockstat` tool (et al.) to point out that it has no tool-specific options?